reporting-app/app/views/staff/certification_batch_uploads/new.html.erb

```diff
@@ -0,0 +1,55 @@
+<% content_for :title, t(".title") %>
+<h1><%= t(".heading") %></h1>
+<p><%= t(".instructions") %></p>
+<p>
+  <%= link_to t(".download_template"), "/certification_batch_upload_template.csv", class: "usa-link", download: true %>
+</p>
+<div class="usa-alert usa-alert--info" role="alert">
+  <div class="usa-alert__body">
+    <h4 class="usa-alert__heading"><%= t(".csv_format.heading") %></h4>
```

Comment:
For accessibility purposes, headers should go in order so they can be navigated using screen readers. They can then be styled to look like an `h4` if that's preferred.

```suggestion
    <h2 class="usa-alert__heading"><%= t(".csv_format.heading") %></h2>
```